xbmc/cores/RetroPlayer/RetroPlayer.cpp

```diff
@@ -187,6 +187,7 @@ bool CRetroPlayer::OpenFile(const CFileItem& file, const CPlayerOptions& options
     RegisterWindowCallbacks();
     SetSpeedInternal(1.0);
     m_callback.OnPlayBackStarted(fileCopy);
+    m_callback.OnAVStarted(fileCopy);
```

Comment:
@garbear could you please have a look. You may want to move OnAVStarted to another pos when av streams are ready. CApp shows a busy dialog the time between OnPlayBackStarted and OnAVStarted. Not sure if this is relevant for RP.

Reply:
In RetroPlayer, A/V streams are independent. Should this callback be called for audio start, video start, or both?

Reply:
it should be called only once. VP fires the event after audio and video streams are ready to go and stream players are unpaused. CApp closes the busy dialog on this event.

Reply:
We could defer this call until the first frame is shown, at the expense of additional complexity. IMO it's not worth it, so placing the callback here is fine.

Reply:
Let's avoid adding player logic to the AV streams. Invoking the callback here is fine.